cohortextractor/tpp_backend.py

```diff
@@ -3491,6 +3491,45 @@ def patients_with_an_ons_cis_record(
             """
         return table_queries + [sql]
 
+    def patients_with_record_in_ukrr(
+        self,
+        # picks dataset held by UKRR
+        from_dataset,
+        returning,
+        # Date filtering: date limits
+        on_or_before=None,
+        on_or_after=None,
+        between=None,
```

Comment:
We only need `between` here - `process_covariate_definitions.handle_time_period_options` converts any `on_or_before` or `on_or_after` args into a between one before it gets to the backend.